riscv32_layout.ld

```diff
@@ -1,7 +1,8 @@
 /* Layout for the RISC-V 32 boards, used by the examples in this repository. */
 
 MEMORY {
-  FLASH (rx) : ORIGIN = 0x20430000, LENGTH = 32M
+  /* The application region is 32 bytes (0x20) */
```

Comment:
TBF header region?

Reply:
Yeah, this comment is just copied from the other *.ld files.